client/modules/IDE/components/Toast.jsx

```diff
@@ -9,19 +9,40 @@ export default function Toast() {
   const { text, isVisible } = useSelector((state) => state.toast);
   const dispatch = useDispatch();
   const { t } = useTranslation();
+
+  useEffect(() => {
+    if (isVisible) {
+      const liveRegion = document.getElementById('toast-live-region');
+      if (liveRegion) {
+        liveRegion.textContent = t(text); // Update live region globally
+      }
+    }
+  }, [isVisible, text, t]);
+
   if (!isVisible) {
     return null;
   }
+
   return (
-    <section className="toast" role="status" aria-live="polite">
-      <p>{t(text)}</p>
-      <button
-        className="toast__close"
-        onClick={() => dispatch(hideToast())}
-        aria-label="Close Alert"
-      >
-        <ExitIcon focusable="false" aria-hidden="true" />
-      </button>
-    </section>
+    <>
+      {/* Global ARIA Live Region */}
+      <div
+        id="toast-live-region"
+        aria-live="polite"
+        style={{ position: 'absolute', left: '-9999px' }}
```

Comment:
We probably want to preserve the original setup here, such as referencing `toast__close` rather than adding the styling here.